docs/specs/shapes.md

```diff
@@ -7,6 +7,37 @@ The graphical elements are divided in 4 categories:
 * [Styles](#shape-style), that define the visual appearance of shapes
 * [Modifiers](#modifier) alter the curves of the shapes
 
+## Shape Rendering Model
+
+The Lottie shape rendering model follows [AfterEffects' Shape Layer](
+https://helpx.adobe.com/after-effects/using/overview-shape-layers-paths-vector.html) semantics.
```

Comment:
I'd rather not reference outside material for this. We need to define our own rendering model

Reply:
Ye could be a note but IMO adobe stuff shouldn't be required reading / part of the specs since it's not a standard